templates/root.html

```
@@ -74,14 +74,16 @@
-    <script src="https://vjs.zencdn.net/7.20.2/video.min.js" integrity="sha384-bvO7eGWph8VhsDHnTORUdWRBA1eMIDM+6dzezVZTlHafSfIguxaRh3lCnUmPNHG7" crossorigin="anonymous"></script>
-    {% else %}
-    <link href="{% static "lib/video-js/7.20.2/video-js.min.css" %}" rel="stylesheet">
-    <script src="{% static "lib/video-js/7.20.2/video.min.js" %}"></script>
-    {% endif %}
+    {% block videojs_assets %}
+        {# Video.js CSS & JS loading #}
+        {% if LOAD_FROM_CDN %}
+        <link href="https://vjs.zencdn.net/7.20.2/video-js.min.css" rel="stylesheet" integrity="sha384-m/+gruJNOZgxOA44GgPxiXxSBPKfJTu36EVTc8fYIVPKd6V8pBhUoCHx/Zl71xQL" crossorigin="anonymous">
+        <script src="https://vjs.zencdn.net/7.20.2/video.min.js" integrity="sha384-bvO7eGWph8VhsDHnTORUdWRBA1eMIDM+6dzezVZTlHafSfIguxaRh3lCnUmPNHG7" crossorigin="anonymous"></script>
+        {% else %}
+        <link href="{% static "lib/video-js/7.20.2/video-js.min.css" %}" rel="stylesheet">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)